test/e2e/run_device_test.go

```diff
@@ -89,12 +89,13 @@ var _ = Describe("Podman run device", func() {
 	})
 
 	It("podman run device host device with --privileged", func() {
-		if _, err := os.Stat("/dev/kvm"); err != nil {
-			Skip("/dev/kvm not available")
-		}
-		session := podmanTest.Podman([]string{"run", "--privileged", ALPINE, "ls", "/dev/kvm"})
+		session := podmanTest.Podman([]string{"run", "--privileged", ALPINE, "ls", "/dev/kmsg"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
+		// verify --privileged is required
+		session2 := podmanTest.Podman([]string{"run", ALPINE, "ls", "/dev/kmsg"})
+		session2.WaitWithDefaultTimeout()
+		Expect(session2).Should(Not(Exit(0)))
```

Comment:
I think the preferred usage is `.To(ExitWithError())` or, even better, `.To(Exit(1))`; but I'll leave it for others to decide how important that is.

Reply:
Should definitely check 1, since this could exit with 125 which would be an unexpected error.

Reply:
oof :confused: using `ls` here is a mistake, it's not a tool designed for this purpose.  I propose we switch to a 'test -c /dev/kmsg' since that's purpose built and will give a consistent exit(1) if the file doesn't exist or isn't a character-device.